admin/src/athletes/Edit.vue

```diff
@@ -20,13 +27,25 @@
           for="kt"
           class="form-label"
         >Kennitala</label>
-        <input
-          id="kt"
-          v-model="currentItem.kt"
-          type="text"
-          class="form-control"
-          :disabled="busy"
-        >
+        <div class="input-group">
+          <input
+            id="kt"
+            v-model="currentItem.kt"
+            type="text"
+            class="form-control"
+            :disabled="busy"
+            @input="event => userId = event.target.value"
```

Comment:
The value of the input is already bound to currentItem.kt, so you don't have to create a new data binding. You can use currentItem.kt

Reply:
OK, done.